pdal/SpatialReference.cpp

```diff
@@ -490,25 +505,18 @@ std::string SpatialReference::getWKT1() const
         return wkt;
 
     OGRScopedSpatialReference srs = ogrCreateSrs(wkt);
-    std::string wkt1;
-    if (srs)
-    {
-        char *buf = nullptr;
-        const char* apszOptions[] =
-            { "FORMAT=WKT1_GDAL", "ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES", nullptr };
-
-        srs->exportToWkt(&buf, apszOptions);
-        if (buf)
-        {
-            wkt1 = buf;
-            CPLFree(buf);
-        }
-    }
+    std::string wkt1 = exportToWkt(srs.get(), {"FORMAT=WKT1_GDAL", "ALLOW_ELLIPSOIDAL_HEIGHT_AS_VERTICAL_CRS=YES"});
     if (wkt1.empty())
-        throw pdal_error("Couldn't convert spatial reference to WKT version 1.");
+       throw pdal_error("Couldn't convert spatial reference to WKT version 1.");
     return wkt1;
 }
 
+std::string SpatialReference::getWKT2() const
+{
+    std::string wkt = getWKT();
+    OGRScopedSpatialReference srs = ogrCreateSrs(wkt);
+    return exportToWkt(srs.get(), {"FORMAT=WKT2"});
```

Comment:
do we want to set this to `WKT2:2019` to be explicit and to be fixed to a single version until we want to update?